edge/pkg/edgehub/config/config.go

```diff
@@ -61,6 +76,11 @@ func init() {
 		log.LOGGER.Errorf("Error in loading Controller configurations in edge hub:  %v", err)
 		panic(err)
 	}
+	err = getQuicConfig()
```

Comment:
the function getQuicConfig doesn't return error. Hardcoded return nil is present. We need to add error returns.

Reply:
On a second thought, if user wants to use websocket, he may not give quic config and vice versa. The check needs to be done basd on protocol used which is given in configuration.